src/taco-cli/cli/remoteBuild/remoteBuildClientHelper.ts

```diff
@@ -67,6 +67,7 @@ class RemoteBuildClientHelper {
 
         var changeTimeFile: string = path.join(settings.platformConfigurationBldDir, "lastChangeTime.json");
         var lastChangeTime: { [file: string]: number } = {};
+        var buildLog: string = path.join(settings.platformConfigurationBldDir, "build.log");
```

Comment:
Perhaps extract this further out and use the same variable in logBuildOutput rather than repeating the logic for constructing it?